src/views/Home.jsx

```diff
@@ -18,12 +22,51 @@ export function Home({ listToken, setListToken }) {
 		}
 	}, [listToken]);
 
+	const handleChange = (e) => {
+		setExistingToken(e.target.value);
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
+		// conects to db
+		// checks if the token that user inputted is associated with any lists
+		streamListItems(existingToken, (snapshot) => {
```

Comment:
Then in `streamListItems` function, we replace `existingToken` with the new variable `caseSensitiveToken` to check in the database if the token exists.
```suggestion
		streamListItems(caseSensitiveToken, (snapshot) => {
```